### ISSUE TITLE
NVDA remote not working with the new update

### ISSUE THREAD

**Issue by liz090300** [NVDARemote/NVDARemote#192]
I updated my nvda today and my remote add on doesn't work anymore! Why?! That was my favorite feature

**Comment by DrSooom** [comment]
@liz090300: If you are using NVDA 2018.3 (aor higher), please update to [NVDA Remote 2.2](https://nvdaremote.com/remote-2.2.nvda-addon).
